### ISSUE TITLE
"Cannot read property 'value' of undefined" on InlineFragment node

### ISSUE THREAD

**Issue by dhritzkiv** [Mikhus/graphql-fields-list#1]
At L209, the code accessing `node.name.value` throws as `node.name` is undefined. This occurs when a node is an inline fragment node

https://github.com/Mikhus/graphql-fields-list/blob/a352157184436ee6cfff69bb9f68631e05aa0cf7/index.ts#L199-L223

The object looks something like this:

```js
node {
    kind: "InlineFragment",
    typeCondition: {/*...*/},
    loc: {/*...*/},
    directives: [],
    selectionSet: {/*...*/}
}
```

A quick workaround that worked for me was to check for the name object on the node, and to continue if it's missing:

```ts
if (!node.name || (opts.withVars && !verifyDirectives(node.directives, opts.vars))) {
   continue;
}
```

but that then ignores the fields in an Inline Fragment.

Another solution is to add this block to inline the inline fragment's node on the current root

```ts
if (node.kind === "InlineFragment") {
    const nodes = getNodes(node);
    nodes.length && traverse(nodes, root, opts);
    continue;
}
```

**Comment by Mikhus** [comment]
Thank you, @dhritzkiv, good catch! I will add Inline fragment checking

**Comment by Mikhus** [comment]
Fixed in v1.1.2, @dhritzkiv, please, verify
Thank you!

**Comment by dhritzkiv** [comment]
Fixed! Thank you very much.
